etna/transforms/timestamp/holiday.py

```diff
@@ -60,19 +93,42 @@ def _get_column_name(self) -> str:
             return self.__repr__()
 
     def _fit(self, df: pd.DataFrame) -> "HolidayTransform":
+        """Fit the transform.
+
+        Parameters
+        ----------
+        ts:
```

Comment:
here we have `df` argument, not `ts`, so lets replace it in docstring.

Reply:
Generally speaking, we don’t need to add this method here, it’s enough just to add `fit` method.